HelpAssistHer/help-assist-her
# Do not list

## Description
- This button is similar to the Out of Business button
- When the button is clicked, the orange overlay should be over the form so that the form cannot be edited.
- The button right below the Out of Business button
## Test Plan
- yarn webpack
- visit localhost/4000
- verify 
      1. Do not list button will appear below the out of business button
      2. Click the button it will bring an orange overlay over form 
      3. Over the form, it will be in the inactive state and not be able to edit
       4. Click again it will remove the overlay now form is in the active state and will be able to edit.
